front/lib/api/assistant/jit_actions.ts

```diff
@@ -340,6 +334,50 @@ export async function renderConversationForModelJIT({
     }
   }
 
+  if (selected.length > 0) {
+    const { filesAsXML, hasFiles } = await listConversationFiles({
+      conversation,
+    });
+    if (hasFiles) {
+      const randomCallId =
+        "list_conversation_files_" + Math.random().toString(36).substring(7);
```

Comment:
The call IDs typically don't have any semantic meaning, not sure the `list_conversation_files` prefix is desirable here

Reply:
I believe, I replicated the one I saw from inspecting conversation with table query (table_query_earjearjk). Will double check, you think it's a problem ?

Reply:
> table_query_earjearjk

Isn't this a function name rather than a function call id ?

Reply:
Indeed, got confused
<img width="432" alt="image" src="https://github.com/user-attachments/assets/b870aa0b-434b-499a-94dd-002965b24ef0">